src/lib/components/chat/ChatMessage.svelte

```diff
@@ -116,7 +116,7 @@
 {#if message.from === "user"}
 	<div class="group relative flex items-start justify-start gap-4 max-sm:text-sm">
 		<div class="mt-5 h-3 w-3 flex-none rounded-full" />
-		<div class="whitespace-break-spaces rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
+		<div class="whitespace-break-spaces break-all rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
```

Comment:
```suggestion
		<div class="whitespace-break-spaces break-words rounded-2xl px-5 py-3.5 text-gray-500 dark:text-gray-400">
```

Reference:

- https://tailwindcss.com/docs/word-break#break-words
- https://tailwindcss.com/docs/word-break#break-all